### PR TITLE
tidb-in-kubernetes: update tidb slow log in cluster operation

### PR DESCRIPTION
**Pull request by DanielZhangQD**
- Related PR: https://github.com/pingcap/docs-cn/pull/1448
- Preview: https://github.com/DanielZhangQD/docs-cn/blob/slow-log/tidb-in-kubernetes/operation-guide.md

### CODE REVIEW

**Comment by lilin90** [comment]
@yikeke Please help resolve the conflicts, review this PR and guarantee these changes are added to the `/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md` file in both `dev` and `v3.0`. Thanks!

**Comment by yikeke** [comment]
> @yikeke Please help resolve the conflicts, review this PR, and guarantee these changes are added to the `/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md` file in both `dev` and `v3.0`. Thanks!

Okay.

File: v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 1]
```diff
@@ -265,36 +265,39 @@ Grafana 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持负
...
+    ```shell
+    kubectl logs -n ${namespace} ${tidbPodName} | grep SLOW_QUERY
+    ```
+    如果 TiDB 版本 >= v2.1.8，由于慢查询日志格式发生变化，不太方便分离慢查询日志，建议参考下面内容配置 `separateSlowLog: true` 单独查看慢查询日志。
 
-```yaml
-# Uncomment the following line to enable separate output of the slow query log
-    # separateSlowLog: true
-```
+* 配置 `separateSlowLog: true` 输出慢查询日志到一个sidecar 容器：
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]:
```suggestion
* 配置 `separateSlowLog: true` 输出慢查询日志到一个 sidecar 容器：
```

File: v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 2]
```diff
@@ -265,36 +265,39 @@ Grafana 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持负
...
 
-运行 `helm upgrade` 使配置生效，然后你可以通过名为 `slowlog` 的 sidecar 容器查看慢查询日志：
+    ```yaml
+    # Uncomment the following line to enable separate output of the slow query log
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]:
```suggestion
```
This line seems unnecessary here, please remove it.

File: v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 3]
```diff
@@ -265,36 +265,39 @@ Grafana 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持负
...
+        separateSlowLog: true
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]:
```suggestion
separateSlowLog: true
```

File: v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 4]
```diff
@@ -265,36 +265,39 @@ Grafana 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持负
 
 ### 查看 TiDB 慢查询日志
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]: Please also update these changes in dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md.

**Pull request merged**